test/support/test_data.ex

```diff
@@ -0,0 +1,53 @@
+defmodule MultipassEx.TestData do
+  @moduledoc false
+  use ExUnitProperties
+
+  @dialyzer :no_return
+
+  def site_key do
+    string(:alphanumeric, min_length: 32)
+  end
+
+  def api_key do
+    string(:alphanumeric, min_length: 32)
+  end
+
+  def example_structure do
+    fixed_map(%{
+      "idk" =>
+        fixed_map(%{
+          "needed" => string(:alphanumeric),
+          "strings" => string(:alphanumeric),
+          "for" => string(:alphanumeric),
+          "this" => string(:alphanumeric),
+          "to" => string(:alphanumeric),
+          "test" => boolean(),
+          "somewhat" => string(:alphanumeric)
+        }),
+      "complex" => list_of(fixed_map(%{})),
+      "structure" => string(:alphanumeric),
+      "with" =>
+        list_of(
+          fixed_map(%{
+            "a" => string(:alphanumeric),
+            "round" => string(:alphanumeric),
+            "trip" => string(:alphanumeric),
+            "of" => boolean()
+          })
+        ),
+      "this" => string(:alphanumeric),
+      "." => boolean()
+    })
+  end
+
+  def pseudo_encrypted_string do
+    string('abcdefghijklmnopqrstuvwxyz+\n/', min_length: 60)
```

Comment:
That's cool you used a charlist to specify the valid characters. 👍 

```
string(Enum.concat([?a..?z, '+\n/'], min_length: 60)
```
might also work, but I'm not sure if if it hurts or improves readability.

Reply:
I had that and changed it due to a PR comment. I could go either way as well on readability.